Libraries/Image/ImageBackground.js

```diff
@@ -67,7 +67,7 @@ class ImageBackground extends React.Component<ImageBackgroundProps> {
 
   render(): React.Node {
     const {children, style, imageStyle, imageRef, ...props} = this.props;
-    const flattenedStyle = flattenStyle(style);
+    const flattenedStyle = flattenStyle(imageStyle);
```

Comment:
The imageStyle is used in line 91 to override the style for Image component. If it has any width or height values, it will be overridden and used anyways. On the other hand, if there's no width/height in imageStyle, this change will cause the width and height not being set in Image component. This will cause issue with loading image and not showing properly inside the image background (See the comments in line 81 to 87 for details).

Reply:
Got it, regarding the issue #33513, how should I specify this percentage?
If I set the percentage directly on imageStyle it looks like this:
<img width="1293" alt="image" src="https://user-images.githubusercontent.com/18408823/161405253-18b92cd7-d2b7-41a6-9513-df7be830c1f5.png">

Reply:
@Marcoo09 btw What is your vs code theme

Reply:
@hetanthakkar1 this one
<img width="915" alt="image" src="https://user-images.githubusercontent.com/18408823/162591116-8c8d322c-f69e-4968-90c1-d8f6d2a903cb.png">